src/hosted-buttons/utils.js

```diff
@@ -77,8 +77,11 @@ export const createAccessToken: CreateAccessToken = memoize<CreateAccessToken>(
   }
 );
 
-const getButtonVariable = (variables: ButtonVariables, key: string): string =>
-  variables?.find((variable) => variable.name === key)?.value ?? "";
+const getButtonVariable = (
+  variables: ButtonVariables,
+  key: string | number
+): string | number =>
+  variables?.find((variable) => variable.name === key)?.value ?? undefined;
```

Comment:
It was returning an empty string before and I had to change it because of the `height` variable.... but this brings up an interesting point!  

I just realized`.find()` returns `undefined` if nothing is found, so we can probably just return `variables.find()` instead of using `?? undefined` right?  I think that's redundant code.

Reply:
I think the original intent was to replace `ButtonVariables` -> `value`s set to `null` or `undefined` with a `""` default value, but the way it's written now, it'll always set a `null` or `undefined` `ButtonVariables` -> `value` to `undefined`. The return type of `getButtonVariable` is now `string | number`, so I don't think that makes sense. What are you wanting the default value to be when `ButtonVariables` -> `value` is set to `null` or `undefined`?

Reply:
I'm thinking the default value can just be undefined! Is there any problem with that or a reason it has to be an empty string? 🤔

Reply:
Ah, okay, so I'm thinking `undefined` should be added to the function return type, which should fix the type error in the screenshot, and then you can skip the nullish coalescing operator if you don't need to replace `null` values with `undefined`.
![Screenshot 2024-04-26 at 12 28 37 PM](https://github.com/paypal/paypal-checkout-components/assets/20399044/5df79b12-4490-4a38-8c29-809de0ab4caf)

Reply:
Yes! Good spot. I added that in 😄  Thank you! 🚀

Reply:
Oh I see why @jshawl originally returned an empty string. Trying to work the types for this is not fun in flow... so it made sense to just always return a string... Though, now if `height` is a number, we have to return `string | number | undefined`... but that's a little tricky in this instance.

Reply:
I guess we _could_ have them send `height` as a string still and then we can convert it to a number on our end 🤔